config-wrapper.js

```diff
@@ -114,7 +114,7 @@ function isValidKeyPath(keyPath) {
 function deepFreezeObject(o) {
     Object.freeze(o);
 
-    Object.getOwnPropertyNames(o).forEach(function eachProp(prop) {
+    Object.keys(o).forEach(function eachProp(prop) {
```

Comment:
if o is a fn this results in callee and other meta properties being iterated. It is an error to access those in strict-mode so the next line throws. These are objects are flattened anyway so i believe Object.keys is quite safe here.